lib/modules/platform/gitlab/index.ts

```diff
@@ -543,9 +543,11 @@ async function ignoreApprovals(pr: number): Promise<void> {
     );
 
     if (existingAnyApproverRule) {
-      await gitlabApi.putJson(`${url}/${existingAnyApproverRule.id}`, {
-        body: { ...existingAnyApproverRule, approvals_required: 0 },
-      });
+      for (const rule of rules) {
+        await gitlabApi.putJson(`${url}/${rule.id}`, {
+          body: { ...existingAnyApproverRule, approvals_required: 0 },
```

Comment:
this seems wrong, you're overwriting the rule with an any approver rule, so you would finally have multiple of them, which isn't allowed on gitlab.

i think we should simply delete all rules when `gitLabIgnoreApprovals=true`